string          joint_names
int64             i_gain
---

